notebooks/Airline Delays - Data Processing Pipeline.py

```diff
@@ -658,6 +677,38 @@ def chain_delay_analysis (dep_time_diff_one_flight_before, dep_time_diff_two_fli
 
 # COMMAND ----------
 
+# Index label
+labelIndexer = StringIndexer(inputCol="dep_del15", outputCol="label")
+
+train_GBT = labelIndexer.fit(train_GBT).transform(train_GBT)
+validation_GBT = labelIndexer.fit(validation_GBT).transform(validation_GBT)
+test_GBT = labelIndexer.fit(test_GBT).transform(test_GBT)
+
+# Index features
+categorical = ["month", "day_of_week", "op_unique_carrier", "Holiday", "PREVIOUS_FLIGHT_DELAYED_FOR_MODELS", "origin_WND_type_code", "origin_CIG_ceiling_visibility_okay", "origin_VIS_variability", "dest_WND_type_code", "dest_CIG_ceiling_visibility_okay", "dest_VIS_variability", "crs_dep_hour"]
+
+categorical_index = [i + "_Index" for i in categorical]
+  
+stringIndexer = StringIndexer(inputCols=categorical, outputCols=categorical_index)
+    
```

Comment:
Same comment as above, only fit on train data
```suggestion
    stringIndexer.fit(train_GBT)
```